src/lib/wagmi.ts

```diff
@@ -84,6 +85,19 @@ export const WAGMI_SUPPORTED_CHAINS: Chain[] = [
   kava,
 ];
 
+export const privyConfig: PrivyClientConfig = {
+  embeddedWallets: {
+    createOnLogin: 'users-without-wallets',
+    requireUserPasswordOnCreate: true,
+    noPromptOnSignature: true,
+  },
+  loginMethods: ['email', 'sms', 'twitter', 'google', 'apple'],
+  appearance: {
+    theme: '#28283c',
+  },
+  defaultChain: sepolia,
```

Comment:
TODO: this will needed to be confirmed based on the network we are on, namely would need to be different for mainnet.

We can get the chainId like this: `Number(ENVIRONMENT_CONFIG_MAP[selectedNetwork].ethereumChainId)`

But might need to be passed in from `App.tsx`